src/datastation/dv_dataset_edit_metadata.py

```diff
@@ -0,0 +1,82 @@
+import csv
+import re
+from argparse import ArgumentParser
+from csv import DictReader
+from os.path import expanduser, isfile
+
+from datastation.common.batch_processing import BatchProcessor
+from datastation.common.config import init
+from datastation.common.utils import add_batch_processor_args, add_dry_run_arg
+from datastation.dataverse.datasets import Datasets
+from datastation.dataverse.dataverse_client import DataverseClient
+
+
+def main():
+    config = init()
+    parser = ArgumentParser(description='Edits one or more, potentially published, datasets. Requires an API token.')
+    parser.add_argument('-r', '--replace', dest="replace", action='store_true',
+                        help="Replace existing metadata fields with the new metadata. ")
+    parser.add_argument('pid_or_file',
+                        help="Either a CSV file or the PID of the dataset to edit. "
+                             "One column column of the file MUST have title 'PID'. "
+                             "The other columns MUST have a typeName, as for the --value argument.")
+    parser.add_argument('-v', '--value', action='append',
+                        help="At least once in combination with a PID, none in combination with a CSV file. "
+                             "The new values for fields formatted as <typeName>=<value>. "
+                             "for example: title='New title'. "
```

Comment:
'typeNme' suggests it is a type, like 'string' or 'intehger', but it is the field 'identifier'? Not sure what is used in the json

Reply:
https://guides.dataverse.org/en/latest/_downloads/cb0fae993277527ea354dc15bc5db864/dataset-edit-metadata-sample.json

Reply:
Ok, so 'typeName' is from the JSON, but the 'user' of this tool does not see that. Something with the word 'field' in it would make more sense, but 'fieldName' is used in the TSV files and there it is not the identifier but the display name. We could replace all 'typeName's by 'field's. Maybe @janvanmansum has a strong opinion about this?